client/pages/car-details.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { Card, Table } from 'react-bootstrap';
+
+class CarDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      carTitle: '',
+      carPhoto: '',
+      carRecords: []
```

Comment:
I think this can just be `car`, no? Like a car object with its properties and one of those properties is an array?

```js
this.state = {
  car: null
};
```

Reply:
Ran into issues when this.state.car was assigned as null.  Instead assigned to an empty object
![image](https://user-images.githubusercontent.com/48267398/163467061-4ab4d0cb-ca05-409e-8af9-8a1f63b1ed9e.png)